source/views/news/news-list.tsx

```diff
@@ -50,11 +52,59 @@ export const NewsList = (props: Props): JSX.Element => {
 		isLoading,
 	} = useNews(props.source)
 
-	// remove all entries with blank excerpts
-	// remove all entries with a <form from the list
-	let entries = data
-		.filter((entry) => entry.excerpt.trim() !== '')
-		.filter((entry) => !entry.content.includes('<form'))
+	let entries = React.useMemo(() => cleanEntries(data), [data])
+
+	let [filters, setFilters] = React.useState<ListType[]>([])
+
+	let getStoryCategories = React.useCallback((story: StoryType) => {
```

Comment:
Question: why not pull this into a standalone function, outside of the component?

Reply:
Funny enough I did pull them outside when I was making the fixes for this last issue. I left as-is to show the minimal diff and then forgot I had lifted them out. Let's pull it out again, that's a good idea!